src/components/Authz/User.scss

```diff
@@ -0,0 +1,7 @@
+.authz-user {
+	max-width: 30ch;
+	white-space: nowrap;
+	word-break: break-word;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
```

Comment:
I would rather remove this file, I think the ellipsis can be done on a particular place (where the component is used)